www/analytics/last-score-by-activity-by-user/index.php

```diff
@@ -0,0 +1,107 @@
+<?php
+$lastScoreBABUReportConfig = [
+    'id' => 'lastscore-activity-by-user',
+    'type' => 'lastscore-by-activity-by-user',
+    'scoring_type' => 'partial',
+    'user_id' => 'mce_student',
+    'display_time_spent' => true,
+    'activities' => [
+        ['id' => 'Weekly_Math_Quiz', 'name' => 'Weekly Math Quiz'],
+        ['id' => 'Summer_Test_1', 'name' => 'Summer Test']
+    ],
+    'users' => [
+        ['id' => 'mce_student', 'name' => 'Jesse Pinkman'],
+        ['id' => 'mce_student_1', 'name' => 'Walter White'],
+        ['id' => 'mce_student_2', 'name' => 'Saul Goodman']
+    ]
+];
+?>
+<style>
+    .lsbabu-button {
+        height: 36px;
+        font-family: 'Gilroy';
+        font-style: normal;
+        font-weight: 700;
+        font-size: 17px;
+        line-height: 100%;
+        text-align: center;
+        border-radius: 5px;
+        border: 1px solid #0071CE ;
+        padding: 8px;
+        margin-right: 16px;
+        margin-bottom: 16px;
+        color: #0071CE;
+        background-color: #FFFFFF;
+    }
+
+    .lsbabu-button:hover {
+        background: #E6F2FC;
+    }
+
+    .lsbabu-button:focus {
+        outline: 2px solid #0071CE;
+        outline-offset: 2px;
+    }
+
+    .lsbabu-button-selected {
+        color: #FFFFFF;
+        background-color: #0071CE;
+    }
+    .lsbabu-button-selected:hover {
+        background: linear-gradient(0deg, rgba(0, 0, 0, 0.2), rgba(0, 0, 0, 0.2)), #0071CE;
+    }
+    .lsbabu-button-selected:active {
+        background: linear-gradient(0deg, rgba(0, 0, 0, 0.2), rgba(0, 0, 0, 0.2)), linear-gradient(0deg, rgba(0, 0, 0, 0.2), rgba(0, 0, 0, 0.2)), #0071CE;
+    }
+    .lsbabu-button-selected:focus {
+        outline: 2px solid #0071CE;
+        outline-offset: 2px;
+    }
+</style>
+
+<!-- Container for the reports api to load into -->
+<h3 id="lastscore-activity-by-user-report"><a href="#lastscore-activity-by-user-report">Last Score by Activity by User Report</a></h3>
+<p>Drill down and see exactly how each student did, per activity. Helpful for identifying specific knowledge or understanding gaps in your group.</p>
+
+<button class="lsbabu-button lsbabu-button-selected" id="lsbabu-default">Default</button>
+<button class="lsbabu-button" id="lsbabu-cutscore1">Cut-score 1</button>
+<button class="lsbabu-button" id="lsbabu-cutscore2">Cut-score 2</button>
+<p>
+    Below shows an example of a Multi cut-score that can be applied on top of the existing report.
+</p>
+
+<?php require 'last-score-by-activity-by-user-default.php' ?>
+<?php require 'last-score-by-activity-by-user-cut-score1.php' ?>
+<?php require 'last-score-by-activity-by-user-cut-score2.php' ?>
+<script>
+const buttons1 = [
```

Comment:
could you change the variable name from `buttons1` to `lastScoreByActivityByUserButtons` or else you feel better show what the buttons do, try to avoid using numbers in the variable name if unnecessary.